### PR TITLE
update pfSense documentation

### PR DESCRIPTION
**Pull request by ellis-elastic**
updated links in the documentation to the vendor documentation.

Type of change

- Enhancement

## What does this PR do?

updated links in the documentation to the vendor documentation.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/pfsense/_dev/build/docs/README.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+
+Currently the integration supports parsing the Firewall, Unbound, DHCP Daemon, OpenVPN, IPsec, and HAProxy logs. All other events will be dropped.
 The firewall, VPN, DHCP, and DNS logs are able to be individually selected via the "Remote Logging Options"
 section within the pfSense settings page.  In order to collect HAProxy or other "package" logs, the "Everything" option
 must be selected. The module is by default configured to run with the `udp` input on port `9001`.
-
-*The HAProxy logs are setup to be compatible with the dashboards from the HAProxy integration.  Install the HAPrxoy integration assets to utilize them.
+<DocCallOut title="Note">
+The HAProxy logs are setup to be compatible with the dashboards from the HAProxy integration.  Install the HAPrxoy integration assets to use them.
+</DocCallOut>
```
Comment (bmorelli25) [inline packages/pfsense/_dev/build/docs/README.md:12]: These notes are specific to our doc build; they don't work in raw markdown. This needs to be converted to a generic note that can be handled by markdown.

**Review by bmorelli25 — CHANGES_REQUESTED** [review]
Looks good, just one change on the note.

File: packages/pfsense/_dev/build/docs/README.md (hunk 1 above)
Comment (ellis-elastic) [reply packages/pfsense/_dev/build/docs/README.md]: Changed

**Review by r00tu53r — APPROVED** [review]
LGTM

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**